templates_jinja2/suggestions/suggest_team_media.html

```diff
@@ -129,15 +129,20 @@ <h4>Existing Videos</h4>
           </div>
           {% endif %}
 
-          {% if medias_by_slugname.grabcad %}
+          {% if medias_by_slugname.grabcad or medias_by_slugname.onshape %}
```

Comment:
Only think is we should add a note that onshape is now a supported format in the copy above in this file

Reply:
Updated. Does this look good?
![submission-description](https://user-images.githubusercontent.com/16738329/60938505-6f227e80-a291-11e9-8ae8-04f54c885ad6.png)